hack/tools/release/notes.go

```diff
@@ -265,6 +256,15 @@ func run() int {
 
 		// if we're doing beta/rc, print breaking changes and hide the rest of the changes
 		if key == warning {
+			if isAlpha(latestTag) {
+				fmt.Printf(warningTemplate, "ALPHA RELEASE")
+			}
+			if isBeta(latestTag) {
+				fmt.Printf(warningTemplate, "BETA RELEASE")
+			}
+			if isRC(latestTag) {
+				fmt.Printf(warningTemplate, "RELEASE CANDIDATE")
+			}
 			if isAlpha(latestTag) || isBeta(latestTag) || isRC(latestTag) {
 				fmt.Printf("<details>\n")
 				fmt.Printf("<summary>More details about the release</summary>\n\n")
```

Comment:
```suggestion
```
At least I assume this is not needed anymore then.

Reply:
Yeah but then you repeat the same 3 times, IMO its just fine as is.

Reply:
Perhaps we can improve the coding style but then lets do it separately since we have similar code convention in other repos as well. Currently I am upto just fixing it and get go with rc release